### PR TITLE
Restriction Peer Selection Update

### PR DESCRIPTION
**Pull request by maurice-daly**
In the documentation it currently states that peer selection can be configured to "local peer discovery". This setting can only be set via MDM based policies in Windows 11 builds, and therefore clarification is required, as although technically the docs state 20H2 and docs here - https://docs.microsoft.com/en-us/windows/deployment/update/waas-delivery-optimization-reference, state 1803, setting the value via MDM policies in these builds will result in a policy failure.

### CODE REVIEW

File: windows/deployment/update/waas-delivery-optimization.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ For information about setting up Delivery Optimization, including tips for the b
...
 - Local Peer Discovery: a new option for **Restrict Peer Selection By** (in Group Policy) or **DORestrictPeerSelectionBy** (in MDM). This option restricts the discovery of local peers using the DNS-SD protocol. When you set Option 2, Delivery Optimization will restrict peer selection to peers that are locally discovered (using DNS-SD). If you also enabled Group mode, Delivery Optimization will connect to locally discovered peers that are also part of the same group (that is, those which have the same Group ID).
+
+**Please Note:** that the "Local peer discovery (DNS-SD)" option can only be set via MDM delivered policies on Windows 11 builds. This feature can be enabled in supported Windows 10 builds by setting the "HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\DeliveryOptimization\DORestrictPeerSelectionBy" value to "2". More information can be found here - https://docs.microsoft.com/en-us/windows/deployment/update/waas-delivery-optimization-reference.
```
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-delivery-optimization.md:43]:
```suggestion
> [!NOTE]
> The Local Peer Discovery (DNS-SD) option can only be set via MDM delivered policies on Windows 11 builds. This feature can be enabled in supported Windows 10 builds by setting the `HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\DeliveryOptimization\DORestrictPeerSelectionBy` value to **2**. For more information, see [Delivery Optimization reference](/windows/deployment/update/waas-delivery-optimization-reference).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @maurice-daly, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by maurice-daly** [comment]
Just checking in to see if this pull request is going to be published?

**Comment by cmknox** [comment]
#sign-off

**Comment by cmknox** [comment]
@JohanFreelancer9 can you help with merging this one. It is related to the other PR I have related to the same information for the Delivery Optimization reference page.

**Comment by JohanFreelancer9** [comment]
@e0i, can you assist to get this merged, please?

**Pull request merged**